### PR TITLE
added metrics for tasks per executor

### PR DESCRIPTION
**Pull request by redixhumayun**
Note: additional files have changed as a result of running `make fmt && cargo clippy --fix --allow-dirty`

### CODE REVIEW

**Comment by maxkozlovsky** [comment]
There is  pub executor_running_task_count: ExecutorRunningTaskCount, field already, I think it is equivalent to the new one.

**Comment by redixhumayun** [comment]
> There is pub executor_running_task_count: ExecutorRunningTaskCount, field already, I think it is equivalent to the new one.

@maxkozlovsky yeah, you're right. I've changed the code. Thanks for pointing it out

File: src/metrics/mod.rs [hunk 1]
```diff
@@ -593,6 +597,23 @@ pub mod coordinator {
...
 
+            let tasks_per_executor = meter
+                .u64_observable_gauge("indexify.coordinator.tasks_per_executor")
+                .with_callback({
+                    let app = app.clone();
+                    move |observer| {
+                        let counts = app.data.indexify_state.executor_running_task_count.inner();
+                        for (executor_id, count) in counts.iter() {
+                            observer.observe(
+                                (*count).try_into().unwrap(),
```
Comment (diptanu) [inline src/metrics/mod.rs:608]: Can we change the data type for executor running task count and not do unwrap here please? Everything else looks fine.

**Review by diptanu — CHANGES_REQUESTED** [review]

File: src/metrics/mod.rs (hunk 1 above)
Comment (redixhumayun) [reply src/metrics/mod.rs]: @diptanu this is done

**Pull request merged**